pkg/controllers/cassandra/nodepool/nodepool.go

```diff
@@ -68,6 +79,61 @@ func (e *defaultCassandraClusterNodepoolControl) removeUnusedStatefulSets(
 	return nil
 }
 
+// Return the lowest int from a slice of ints
+func min(v []int) (m int) {
+	if len(v) > 0 {
+		m = v[0]
+	} else {
+		panic("min called with empty slice")
+	}
+	for _, e := range v {
+		if e < m {
+			m = e
+		}
+	}
+	return
+}
+
+func (e *defaultCassandraClusterNodepoolControl) labelSeedNodes(
+	cluster *v1alpha1.CassandraCluster,
+	set *appsv1beta1.StatefulSet,
+) error {
+	var nodepoolPods []*v1.Pod
+	allPods, err := e.pods.Pods(cluster.Namespace).List(labels.Everything())
+	if err != nil {
+		return err
+	}
+	for _, pod := range allPods {
+		if metav1.IsControlledBy(pod, set) {
+			nodepoolPods = append(nodepoolPods, pod)
+		}
+	}
```

Comment:
:grey_question: This should work, but we also add nodepool labels to the pods, so it might also be possible to supply a more specific label selector to the lister, above.